dash-renderer/src/components/error/CallbackGraph/CallbackGraphContainer.react.js

```diff
@@ -5,13 +5,22 @@ import stylesheet from './CallbackGraphContainerStylesheet';
 import ReactJson from 'react-json-view'
 
 import Cytoscape from 'cytoscape';
-import Dagre from 'cytoscape-dagre';
 import CytoscapeComponent from 'react-cytoscapejs';
-Cytoscape.use(Dagre);
 
 import PropTypes from 'prop-types';
 
 
+// Set the layout method.
+// NOTE: dagre should be nicer for DAG, but even with manual
+//       cycle pruning, it gives terrible results with State.
```

Comment:
@jjaraalm can you share some examples where dagre doesn't look good? I'm testing a few things and they generally look good to me - and much better than breadthfirst.